alert.go

```diff
@@ -336,6 +354,13 @@ func newAlertNode(et *ExecutingTask, n *pipeline.AlertNode, l *log.Logger) (an *
 		}
 		an.levels[CritAlert] = statefulExpression
 		an.scopePools[CritAlert] = stateful.NewScopePool(stateful.FindReferenceVariables(n.Crit.Expression))
```

Comment:
Yeah, at first I thought the reset expression is kind of "opposite" to the normal expression, so it may use the exact fields and tags from the scope pool of the normal expression. I'll have this updated.